chapters/en/chapter6/pre-trained_models.mdx

```diff
@@ -183,6 +183,94 @@ pre-trained checkpoint only supports English language:
 	height="450">
 </iframe>
 
+## Bark
+
+Bark is a transformer-based text-to-speech model proposed by Suno AI in [suno-ai/bark](https://github.com/suno-ai/bark).
+
+Bark is made of 4 main models:
```

Comment:
A diagram would be nice here! Especially for course material. No strong need to do it now, but in the future it's worth considering when the architecture gets more complicated (see https://huggingface.co/learn/audio-course/chapter7/voice-assistant)